capsules/extra/src/net/thread/thread_utils.rs

```diff
@@ -0,0 +1,282 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
```

Comment:
```suggestion
// Copyright Tock Contributors 2023.
```

Here and elsewhere :)